app/Database/Schema/AccountSchema.php

```diff
@@ -40,10 +40,9 @@ trait AccountSchema
     protected $subscriptionType;
 
     /**
-     * @ORM\ManyToOne(targetEntity="User", inversedBy="account")
-     * @ORM\JoinColumn(nullable=false)
-     * 
-     * @var \App\Database\Schema\User
+     * @ORM\Column(type="string", name="user_id")
```

Comment:
Type should be `guid` since we use this as foreign key to `users.id` which has `guid` type.